utils/errors.ts

```diff
@@ -1,9 +1,18 @@
+import * as Sentry from "@sentry/nextjs";
+import { ContentfulSynth } from "./contentful";
+
 export function errorFilter(value: unknown | Error): boolean {
   if (value instanceof Error) {
-    // TODO: Have a proper error reporting mechanism
-    console.log(`Errored at`);
-    console.log(value);
+    Sentry.captureException(value);
```

Comment:
where does `errorFilter` get called?

Reply:
It gets called in `index.tsx` and `[address].tsx`  after a map with `fetchCompleteSynth` to report the synths that errored out because misconfigured in the CMS and keep the ones configured correctly